### PR TITLE
Adding units for coefficient of thermal expansion

### PR DESCRIPTION
**Pull request by tmilnthorp**

### CODE REVIEW

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "Units": [
+    {
+      "SingularName": "InverseKelvin",
+      "PluralName": "InverseKelvin",
+      "FromUnitToBaseFunc": "x",
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "K⁻¹" ]
```
Comment (angularsen) [inline Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: Chrome on mobile is not reading this symbol right. Is the file utf8 encoded?

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "Name": "CoefficientOfThermalExpansion",
+  "BaseUnit": "InverseKelvin",
+  "XmlDoc": "A unit that represents a fractional change in size in response to a change in temperature.",
+  "BaseDimensions": {
+    "Θ": -1
+  },
+  "Units": [
+    {
+      "SingularName": "InverseKelvin",
+      "PluralName": "InverseKelvin",
```
Comment (angularsen) [inline Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: InverseKelvins?

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      "FromBaseToUnitFunc": "x",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "K⁻¹" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "InverseCelsius",
```
Comment (angularsen) [inline Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: InverseDegreeCelsius and InverseDegreesCelius?
Matches Temperature and TemperatureDelta. 

https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/TemperatureDelta.json

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json (hunk 1 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: Odd. I see the same on my mobile. Looks fine on the web and in Visual Studio. Yes it is UTF-8. It's a superscript -1.

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json (hunk 2 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: I think we should leave them as singular. Coefficients of thermal expansion can be though of as a fractional change "per Kelvin" (K⁻¹ or 1/K).

The unit is always in relation to a change of 1 K, °C, or °F.

File: Common/UnitDefinitions/CoefficientOfThermalExpansion.json (hunk 3 above)
Comment (tmilnthorp) [reply Common/UnitDefinitions/CoefficientOfThermalExpansion.json]: See the reply above about the singular/plural naming. However I agree adding Degree is a good idea. Consistency is always good!

**Comment by angularsen** [comment]
Released in https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F3.107.1

**Pull request merged**